client-app/pages/account/profile.vue

```diff
@@ -191,43 +187,64 @@ const { value: oldPassword } = useField<string>("oldPassword");
 const { value: newPassword } = useField<string>("newPassword");
 const { value: confirmNewPassword } = useField<string>("confirmNewPassword");
 
-const onSubmit = handleSubmit(async (data) => {
-  const results: boolean[] = [];
+const commonErrors = ref<string[]>([]);
+const showErrors = (responseErrors: IdentityErrorType[]) => {
+  responseErrors.forEach((error) => {
+    let errorDescription;
+    if (error.code === "PasswordMismatch") {
+      errorDescription = t("shared.account.change_password_form.errors.wrong_current_pass");
+    } else {
+      errorDescription = getIdentityErrorTranslation(error);
+    }
+
+    if (errorDescription) {
+      commonErrors.value.push(errorDescription);
+    }
+  });
+};
```

Comment:
This duplicated code should probably be moved into composable

Reply:
Great Idea! I'll make a task

Reply:
Keep standrad